HW03/Danila_Glazov/danila_glazov_hw_03_t_04.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: false
+
+DIGITS = /[^\d]/.freeze
+LETTERS = /[^a-zA-Z]/.freeze
+
+def string_info(str)
+  digits = str.gsub(DIGITS, '')
+  letters = str.gsub(LETTERS, '')
```

Comment:
Interesting approach 👍 But could you use a more clear naming? For example
```suggestion
NOT_DIGITS = /[^\d]/.freeze
NOT_LETTERS = /[^a-zA-Z]/.freeze

def string_info(str)
  digits = str.gsub(NOT_DIGITS, '')
  letters = str.gsub(NOT_LETTERS, '')
```